src/main/steps/common/refuge/keeping-details-safe/content.ts

```diff
@@ -81,22 +81,22 @@ export const generateContent = (content: CommonContent): PageContent => {
     ...translations,
     title: C100rebuildJourney
       ? interpolate(translations.c100.title, {
-          name: `${c100Person.firstName} ${c100Person.lastName}`,
+          name: `${c100Person?.firstName} ${c100Person?.lastName}`,
```

Comment:
if there is no other person does it display undefined or blank for names?

Reply:
It displays undefined